packages/batch-poster-monitor/index.ts

```diff
@@ -161,21 +162,60 @@ const showAlert = (childChainInformation: ChainInfo, reasons: string[]) => {
   )
 }
 
+type EventLogs = Log<
+  bigint,
+  number,
+  false,
+  AbiEvent,
+  undefined,
+  [AbiEvent],
+  string
+>[]
+
+const getBatchPosterFromEventLogs = async (
+  eventLogs: EventLogs,
+  parentChainClient: PublicClient
+) => {
+  // get the batch-poster for the first event log
+  const batchPostingTransactionHash = eventLogs[0].transactionHash
+  const tx = await parentChainClient.getTransaction({
+    hash: batchPostingTransactionHash,
+  })
+  return tx.from
+}
+
 const getBatchPosterLowBalanceAlertMessage = async (
   parentChainClient: PublicClient,
-  childChainInformation: ChainInfo
+  childChainInformation: ChainInfo,
+  sequencerInboxLogs: EventLogs
 ) => {
-  //@ts-ignore - PublicClient that we pass vs PublicClient that orbit-sdk expects is not matching
-  const { batchPosters } = await getBatchPosters(parentChainClient, {
-    rollup: childChainInformation.ethBridge.rollup as `0x${string}`,
-    sequencerInbox: childChainInformation.ethBridge
-      .sequencerInbox as `0x${string}`,
-  })
-  if (!batchPosters || batchPosters.length === 0) {
-    return `Batch poster information not found`
+  let batchPoster: `0x${string}` | null = null
+
+  // try fetching batch poster address from orbit-sdk
+  try {
+    //@ts-ignore - PublicClient that we pass vs PublicClient that orbit-sdk expects is not matching
+    const { batchPosters } = await getBatchPosters(parentChainClient, {
```

Comment:
Wrapped the original method in try-catch, and if it fails (eg. for core chains) then we try fetching the batch poster from event-logs.